src/App/App.js

```diff
@@ -44,60 +47,72 @@ export class App extends Component {
     })
   }
 
-  toggleView = (event) => {
-    event.preventDefault()
-    if (this.state.isSingleView) {
-      this.setState((prevState) => {
-        return { isSingleView: !prevState.isSingleView }
-      })
-    } else if (!this.state.isSingleView && event.target.name !== 'home') {
-      this.setState((prevState) => {
-        this.fetchData(`movies/${event.target.id}`)
-          .then((data) => {
-            this.setState({
-              singleMovie: data.movie,
-              isLoading: false,
-              isSingleView: !prevState.isSingleView
-            })
-          })
-          .catch((error) => this.setState({ isError: true }))
-      })
-    }
+  toggleSingleView = () => {
+    this.setState((prevState) => {
+      return { isSingleView: !prevState.isSingleView }
+    })
   }
 
   render() {
     return (
-      <div>
-        <Header
-          movies={this.state.movies}
-          isSingleView={this.state.isSingleView}
-          toggleView={this.toggleView}
-          filterMovies={this.filterMovies}
+      <Switch>
+        <Route
+          exact
+          path='/'
+          render={() => {
+            return (
+              <div>
+                <Header
+                  movies={this.state.movies}
+                  isSingleView={this.state.isSingleView}
+                  filterMovies={this.filterMovies}
+                />
+                {this.state.isError && (
+                  <h1 style={{ textAlign: 'center', marginTop: '5vh' }}>
```

Comment:
Because there is some repetition in the styling here, why use inline styling? Why not apply a className to the element?

Reply:
No real reason, just added the inline styling while moving through the code. We will add a className and apply normal styling when refactoring!